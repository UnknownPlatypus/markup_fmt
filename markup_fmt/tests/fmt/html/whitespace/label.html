<div>
  <label></label>
  <input>
</div>
