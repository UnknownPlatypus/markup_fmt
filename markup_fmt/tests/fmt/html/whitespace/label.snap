---
source: markup_fmt/tests/fmt.rs
---
<div>
  <label></label>
  <input>
</div>
